sshd/issue_invite.sh

```diff
@@ -67,18 +70,13 @@ cat $TMP/id_rsa.pub >> $HOMEDIR/.ssh/authorized_keys
 
 # Output the actual invite code.
 PUBLIC_IP=`cat /hostname`
-export CLOUD_INSTANCE_DETAILS=$(cat << EOF
-{
-  "host":"$PUBLIC_IP",
-  "user":"$USERNAME",
-  "key":"$ENCODED_KEY"
-}
-EOF
-)
+export CLOUD_INSTANCE_DETAILS="{\"host\":\"$PUBLIC_IP\",\"user\":\"$USERNAME\",\"key\":\"$ENCODED_KEY\"}"
 
-echo|base64 -w 0 << EOF
-{
-  "networkName":"Cloud",
-  "networkData":"`echo -n $CLOUD_INSTANCE_DETAILS|sed s/'"'/'\\\\"'/g`"
-}
-EOF
+if [ "$COMPLETE" = true ]
+then
+  npm install jsurl &>/dev/null
+  CLOUD_INSTANCE_DETAILS=`nodejs -p -e "require('jsurl').stringify('$CLOUD_INSTANCE_DETAILS');"`
+  echo "https://www.uproxy.org/invite/v=2&networkName=Cloud&networkData=$CLOUD_INSTANCE_DETAILS"
```

Comment:
`?` to denote start of parameters:
`echo "https://www.uproxy.org/invite?v=2&networkName=Cloud&networkData=$CLOUD_INSTANCE_DETAILS"`